docs/conf.py

```diff
@@ -54,7 +54,9 @@ def noop_decorator(*args, **kwargs):
 if not hasattr(sys.modules["unidist"].core.base, "object_ref"):
     sys.modules["unidist"].core.base.object_ref = type("object_ref", (object,), {})
 if not hasattr(sys.modules["unidist"].core.base.object_ref, "ObjectRef"):
-    sys.modules["unidist"].core.base.object_ref.ObjectRef = type("ObjectRef", (object,), {})
+    sys.modules["unidist"].core.base.object_ref.ObjectRef = type(
```

Comment:
Why is this changed? Should be reverted.

Reply:
Done